content/client-server-api/modules/mentions.md

```diff
@@ -3,6 +3,9 @@
 
 {{% changed-in v="1.7" %}}
 
+{{% changed-in v="1.16" %}}: the legacy push rules that looked for mentions in
```

Comment:
Oops, I forgot about this and the next spec version changed in the meantime (I think 1.17 is the correct value, the release process will presumably `s/1.17/2.0/g` in case the next version ends up being 2.0). Otherwise looks good

Reply:
I updated the spec version.